src/customer-service/Actions/ShowSingleIncidentAction.cs

```diff
@@ -0,0 +1,85 @@
+﻿using customer_service.Models;
+using System;
+using System.Globalization;
+
+namespace customer_service
+{
+    /**
+     * Class: ShowSingleIncidentAction
+     * Purpose: Displays the menu and details for the single incident report
+     * Author: Dayne Wright
+     * Methods:
+     *     void ReadInput() - Shows the incident report and prompts for a resolution if one is not provided. 
+     *     Then saves that resolution with the current date.
+     */
+
+    public class ShowSingleIncidentAction
+    {
+        /**
+         * Purpose: Shows incident report and saves resolution if needed.
+         * Arguments:
+         *     NONE
+         * Return:
+         *     Void
+         */
+        public static void ReadInput()
+        {
+            Customer customer = CustomerFactory.Instance.ActiveCustomer;
+            Incident incident = IncidentFactory.Instance.ActiveIncident;
+
+            if (customer == null || incident == null)
+            {
+                Console.WriteLine("A customer or incident could not be found.");
+                Console.Write("> ");
+                Console.ReadLine();
+            }
+            else
+            {
+                LabelFactory labelFactory = new LabelFactory();
+                IncidentTypeFactory typesFactory = new IncidentTypeFactory();
+
+                var labels = labelFactory.GetLabels(incident.IncidentTypeId);
```

Comment:
This isn't working like it should. It grabs all Labels instead of only the Labels applicable to the IncidentType. I'll open a separate Issue.